changelog.md

```diff
@@ -1,90 +0,0 @@
-# Changelog
```

Comment:
We had a quick discussion on cms-squad slack channel about this.  I mentioned the module changelog is massively out of date - the last release mentioned is 2.2.0 meanwhile this module is up to 3.6.2.  We agreed that we should just prune these things since they do more harm than good being so out of date.

Our actual changelog is https://docs.silverstripe.org/en/4/changelogs/4.5.2/

If you think this is wrong and we should have up-to-date changelogs on all the modules as well then that's fine, I can undelete this changelog, though that'll be a separate issue from this pull-request

Reply:
Hmm yeah it's a bit out of context (I'm not on that channel). So if you slip in changes like that in a PR, it's good to explain why. The best way to do that is through an isolated commit removing the changelog. I don't think now is the time to kick up a discussion about maintaining module specific changelogs, since we maintain recipes rather than modules. So tend to agree with the change itself.